spec.bs

```diff
@@ -506,8 +507,8 @@ To <dfn>validate and convert auction ad config</dfn> given an {{AuctionAdConfig}
     * |directFromSellerSignalsPrefix|'s [=url/query=] is not null.
   1. [=Assert=]: |directFromSellerSignalsPrefix|'s [=url/scheme=] is "`https`".
 1. If |config|["{{AuctionAdConfig/sellerTimeout}}"] [=map/exists=], set |auctionConfig|'s
-  [=auction config/seller timeout=] to min(|config|["{{AuctionAdConfig/sellerTimeout}}"], 500)
-  milliseconds.
+  [=auction config/seller timeout=] to |config|["{{AuctionAdConfig/sellerTimeout}}"] in milliseconds
+  or 500 milliseconds, whichever is smaller.

```

Comment:
I'd probably phrase this as:

```suggestion
  [=auction config/seller timeout=] to the smaller of |config|["{{AuctionAdConfig/sellerTimeout}}"] milliseconds
  or 500 milliseconds.
```

But your phrasing, or even the original min() use, are clear enough.

Reply:
the spec review doc commented that min(xx) is not a spec syntax and suggested the new wording.
I'll keep the "whichever is smaller" phrase since it's not that different.